synapse/cores/common.py

```diff
@@ -2536,27 +2384,7 @@ def incTufoProp(self, tufo, prop, incval=1):
         return self._incTufoProp(tufo, prop, incval=incval)
 
     def _incTufoProp(self, tufo, prop, incval=1):
-
-        # to allow storage layer optimization
-        iden = tufo[0]
-
-        form = tufo[1].get('tufo:form')
-        valu = tufo[1].get(form)
-
-        with self.inclock:
-            rows = self._getRowsByIdProp(iden, prop)
-            if len(rows) == 0:
-                raise s_common.NoSuchTufo(iden=iden, prop=prop)
-
-            oldv = rows[0][2]
-            newv = oldv + incval
-
-            self.setRowsByIdProp(iden, prop, newv)
-
-            tufo[1][prop] = newv
-            self.fire('node:prop:set', form=form, valu=valu, prop=prop, newv=newv, oldv=oldv, node=tufo)
-
-        return tufo
+        return self.store._incTufoProp(tufo, prop, incval=incval)
```

Comment:
might be more consistent to leave this API in the cortex layer since it's aware of tufos